charts/cluster/templates/initial-backup.yaml

```diff
@@ -0,0 +1,17 @@
+{{ if .Values.backups.enabled }}
+apiVersion: postgresql.cnpg.io/v1
+kind: Backup
+metadata:
+  name: {{ include "cluster.fullname" . }}-initial-backup-{{now | unixEpoch}}
+  annotations:
+    "helm.sh/hook": post-install
```

Comment:
Enabling backups after initial install won't create an initial backup.

Best solution, though nog argoCD compatible, would be to do a lookup and skip creating a backup if one with said name already exists combined with helm.sh/resource-policy: keep to prevent deletion using helm.